src/TextLabel/TextLabel.tsx

```diff
@@ -2,16 +2,17 @@ import * as React from "react";
 import "./text-label-style.scss";
 
 export interface TextLabelProps {
-    value: string | number;
-    name?: string;
-    label?: string;
     className?: string;
+    id?: string;
+    label?: string;
+    name?: string;
```

Comment:
This is not being used anymore. Did you mean to remove it?

Reply:
fixed in e90bd8f73c3e2216a90ccb30713b1707f1241063